test/integration_test/common_test.go

```diff
@@ -559,6 +576,125 @@ func scheduleClusterPair(ctx *scheduler.Context, skipStorage, resetConfig bool,
 	return nil
 }
 
+// Create a cluster pair from source to destination and another cluster pair from destination to source
+func scheduleBidirectionalClusterPair(cpName, cpNamespace string) error {
+	// Setting kubeconfig to source because we will create bidirectional cluster pair based on source as reference
+	err := setSourceKubeConfig()
+	if err != nil {
+		return fmt.Errorf("during cluster pair setting kubeconfig to source failed %v", err)
+	}
+
+	// Create namespace for the cluster pair on source cluster
+	_, err = core.Instance().CreateNamespace(&v1.Namespace{
+		ObjectMeta: meta.ObjectMeta{
+			Name: cpNamespace,
+			Labels: map[string]string{
+				"creator": "stork-test",
+			},
+		},
+	})
+	if err != nil {
+		fmt.Errorf("Failed to create namespace %s on source cluster", cpNamespace)
+	}
+
+	// Create directory to store kubeconfig files
+	err = os.MkdirAll(path.Join(specDir, bidirectionalClusterPairDir), 0777)
+	if err != nil {
+		logrus.Errorf("Unable to make directory (%v) for cluster pair spec: %v", specDir+"/"+bidirectionalClusterPairDir, err)
+		return err
+	}
+	srcKubeconfigPath := path.Join(specDir, bidirectionalClusterPairDir, "src_kubeconfig")
+	srcKubeConfig, err := os.Create(srcKubeconfigPath)
+	if err != nil {
+		logrus.Errorf("Unable to write source kubeconfig file: %v", err)
+		return err
+	}
+
+	// Dump source config to the directory created before
+	err = dumpKubeConfigPath(srcConfig, srcKubeconfigPath)
+	if err != nil {
+		return fmt.Errorf("unable to dump remote config while setting source config: %v", err)
+	}
+	defer func() {
```

Comment:
Move this to line 613 so that you always close the file if `dumpKubeConfigPath` also fails

Reply:
Fixed.